### PR TITLE
Working tests in Travis.

### PR DESCRIPTION
**Pull request by demiankatz**
Tests are working!

This may be a bit fragile for two reasons:

1.) Travis doesn't support Chrome by default, and the new container infrastructure doesn't make it easy to install, so we have to download the compiled binary from a fixed location.

2.) This binary distribution has problems with the sandbox, so we have to use the --no-sandbox switch (bad security practice, but in this context I don't think we care). I tried to work around this in other ways but didn't find an easy fix.

Bottom line: it works, but it might eventually stop working. I figure we can cross that bridge when we come to it!

### CODE REVIEW

**Comment by edsilv** [comment]
Awesome!

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,16 @@ node_js:
...
 - npm install
 - bower install
+before_script:
+- ./node_modules/protractor/bin/webdriver-manager update
+- ./node_modules/protractor/bin/webdriver-manager start &
 script:
-- grunt build
\ No newline at end of file
+- grunt build
+- grunt serve &
```
Comment (edsilv) [inline .travis.yml]: Just out of interest, what's the deal with the trailing ampersands? (line 16 too)

Comment (demiankatz) [inline .travis.yml]: These cause the processes to run in the background. I believe that if we omitted them, the build would just hang forever while the server processes waited for input. (Haven't tried it, though).
